src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -78,10 +82,99 @@ public AuditUtility(
+
+            if (packagesWithKnownVulnerabilities == null)
+            {
+                return;
+            }
+
+            PackageDownloadPackagesWithAdvisory = new(capacity: packagesWithKnownVulnerabilities.Values.Count);
+
+            // no-op checks DGSpec hash, which means the order of everything must be deterministic.
+            foreach ((PackageIdentity package, PackageAuditInfo auditInfo) in packagesWithKnownVulnerabilities.OrderBy(p => p.Key.Id))
+            {
+                PackageDownloadPackagesWithAdvisory.Add(package.Id);
+
+                foreach (var advisory in auditInfo.GraphsPerVulnerability.Keys)
+                {
+                    PackageVulnerabilitySeverity severity = advisory.Severity;
+                    if (severity == PackageVulnerabilitySeverity.Low) { Sev0DirectMatches++; }
+                    else if (severity == PackageVulnerabilitySeverity.Moderate) { Sev1DirectMatches++; }
+                    else if (severity == PackageVulnerabilitySeverity.High) { Sev2DirectMatches++; }
+                    else if (severity == PackageVulnerabilitySeverity.Critical) { Sev3DirectMatches++; }
+                    else { InvalidSevDirectMatches++; }
+                }
+            }
+        }
+
+        private Dictionary<PackageIdentity, PackageAuditInfo>? FindDownloadDependenciesWithKnownVulnerabilities(
+            IReadOnlyList<IReadOnlyDictionary<string, IReadOnlyList<PackageVulnerabilityInfo>>> knownVulnerabilities)
+        {
+            Dictionary<PackageIdentity, PackageAuditInfo>? result = null;
+
+            foreach (var downloadDependency in _downloadDependencies)
+            {
+                List<PackageVulnerabilityInfo>? knownVulnerabilitiesForPackage = GetKnownVulnerabilities(downloadDependency.Name, downloadDependency.VersionRange.MaxVersion!, knownVulnerabilities);
+
+                if (knownVulnerabilitiesForPackage?.Count > 0)
+                {
+                    PackageIdentity packageIdentity = new(downloadDependency.Name, downloadDependency.VersionRange.MinVersion);
+
+                    foreach (PackageVulnerabilityInfo knownVulnerability in knownVulnerabilitiesForPackage)
+                    {
+                        if ((int)knownVulnerability.Severity < (int)MinSeverity && knownVulnerability.Severity != PackageVulnerabilitySeverity.Unknown)
+                        {
+                            continue;
+                        }
+
+                        if (SuppressedAdvisories?.TryGetValue(knownVulnerability.Url.OriginalString, out bool advisoryUsed) == true)
+                        {
+                            TotalWarningsSuppressedCount++;
+
+                            if (!advisoryUsed)
+                            {
+                                SuppressedAdvisories[knownVulnerability.Url.OriginalString] = true;
```

Comment:
@zivkan Do we think we'd want to different between PD and PR suppressions?
I feel like we would, mostly because PD could be indicative of out of date SDK and that's a totally different problem.

Reply:
I have removed this telemetry as well. It does not help with the question we are trying to answer